packages/xchain-litecoin/src/node-api.ts

```diff
@@ -12,20 +12,18 @@ import { TxBroadcastResponse } from './types/node-api-types'
  */
 export const broadcastTx = async ({ txHex, auth, nodeUrl }: BroadcastTxParams): Promise<string> => {
   const uniqueId = new Date().getTime().toString() // for unique id
-  const response: TxBroadcastResponse = (
-    await axios.post(
-      nodeUrl,
-      {
-        jsonrpc: '2.0',
-        method: 'sendrawtransaction',
-        params: [txHex],
-        id: uniqueId,
-      },
-      {
-        auth,
-      },
-    )
-  ).data
+  const postData = {
+    jsonrpc: '2.0',
+    method: 'sendrawtransaction',
+    params: [txHex],
+    id: uniqueId,
+  }
+  let response: TxBroadcastResponse
+  if (auth) {
+    response = (await axios.post(nodeUrl, postData, { auth })).data
```

Comment:
If auth is falsy, won't axios just omit it anyway? If so, you can get rid of the if statement; if not, though, this is fine, so I'll approve preemptively.